msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -154,6 +157,9 @@
     @SerializedName(POWER_OPT_CHECK_FOR_NETWORK_REQUEST_ENABLED)
     private Boolean powerOptCheckEnabled;
 
+    @SerializedName(HANDLE_TASKS_WITH_NULL_TASKAFFINITY)
+    private Boolean handleNullTaskAffinity;
```

Comment:
```suggestion
    private Boolean mHandleNullTaskAffinity;
```

`m` prefix desired? (other changes needed in this file if you take this suggestion)

Reply:
again...we didn't do that above... happy to make them all consistent.

Reply:
Hm. yes -- your call, consistency might be good but no need to go disturbing the force if you perceive any risk or that this is unneeded churn